src/js/components/ChatFoot/ChatFoot.js

```diff
@@ -72,10 +77,7 @@ const ChatInput = (props) => {
       return;
     }
 
-    setInputState({
-      ...inputState,
-      currentInputHeight: scrollHeight,
-    });
+    setChatInputState(id, chatInput, scrollHeight);
```

Comment:
Why force a rerender? Don't you have enough information to just render the component as needed?